src/helpers/getAssetSettingsFromToml.ts

```diff
@@ -56,3 +58,22 @@ export const getAssetSettingsFromToml = async ({
     ...{ supportedActions },
   };
 };
+
+const isSEP8Asset = ({
```

Comment:
Super nitpicky, we have a convention of using "pure" camel case convention (not having SEP in all caps, for example), so this function would be `isSep8Asset`. 😊

Reply:
SGTM! Updated that in https://github.com/stellar/stellar-demo-wallet/pull/151/commits/c016d43e5fe254cb10c46e90a5e323555c3f33dc.